packages/eslint-config-vivy/index.js

```diff
@@ -16,6 +16,16 @@ module.exports = {
       { extensions: [".js", ".jsx", ".tsx"] },
     ],
     "prettier/prettier": ["error"],
+    "react/jsx-props-no-spreading": 1,
```

Comment:
can we avoid adding extra rules that might break existing projects for now? or is there a reason for this change?

Reply:
I was just fixing errors that came up in other projects. will revert any changes to the rules 👍